opts/opts.go

```diff
@@ -188,3 +188,35 @@ func ValidateLabel(val string) (string, error) {
 	}
 	return val, nil
 }
+
+// ValidateSysctl validates an sysctl and returns it.
+func ValidateSysctl(val string) (string, error) {
+	validSysctlMap := map[string]bool{
+		"kernel.msgmax":          true,
```

Comment:
hey @rhatdan,

so i ran the container with the `--privileged` flag as such (as you can see, i'm on a mac; unsure if this matters to this experiment):

````
docker run -d -p 8080:8080 --name foobar -v /Users/rlune/gitproj/observable_node_poc:/src -v /Users/rlune/gitproj/observable_node_poc:/tmp --privileged observable-node-poc
````
... and got:

````
d137a82d7731ab095c484d0e70dd96081c275cf43c50100128ad5ea44e40cb01
````

... then, `exec`'d a `bash` session:

````
docker exec -it d137a82d7731ab095c484d0e70dd96081c275cf43c50100128ad5ea44e40cb01 bash
````
... checked the host machine for the kernel flags next...:

````
16:24 rlune@oakm111430f01:~/gitproj/observable_node_poc $ sysctl -a | grep kernel.perf_event_paranoid
16:25 rlune@oakm111430f01:~/gitproj/observable_node_poc $ sysctl -a | grep kernel.kptr_restrict
````
... received no output for each lookup. then, set the flags on the instance running in the container:

````
[root@d137a82d7731 app]# sysctl -w kernel.kptr_restrict=1
kernel.kptr_restrict = 1
[root@d137a82d7731 app]# sysctl -w kernel.perf_event_paranoid=1
kernel.perf_event_paranoid = 1
````

... read the keys on the image:

````
[root@d137a82d7731 app]# sysctl -a | grep kernel.kptr_restrict      
kernel.kptr_restrict = 1
sysctl: reading key "net.ipv6.conf.all.stable_secret"
sysctl: reading key "net.ipv6.conf.default.stable_secret"
sysctl: reading key "net.ipv6.conf.eth0.stable_secret"
sysctl: reading key "net.ipv6.conf.lo.stable_secret"

[and got the same sort of output for the `kernel.perf_event_paranoid` also]
````

... then checked the host machine again:

````
16:24 rlune@oakm111430f01:~/gitproj/observable_node_poc $ sysctl -a | grep kernel.perf_event_paranoid
16:25 rlune@oakm111430f01:~/gitproj/observable_node_poc $ sysctl -a | grep kernel.kptr_restrict
````

... and did not see any lookup output for those keys.

is this test sufficient to determine that the `kernel.perf_event_paranoid` and `kernel.kptr_restrict` keys are namespaced in a way that will allow their addition to this PR?